helpers/compile/build.ts

```diff
@@ -144,6 +144,8 @@ async function dependencyCheck(options: BuildOptions) {
  * @param options
  */
 export async function build(options: BuildOptions[]) {
+  if (process.env.ECOSYSTEM === 'true') return
```

Comment:
When we trigger `pnpm pack` we actually don't want to build again to go faster. We re-use what has been already build; and also implies you ran `pnpm run watch/dev/build` before hand. In the future, I think I'll enable a full build + packaging mode to be much closer to reality, this could be done on CI.